internal/service/advancedclustertpf/validate_schema.go

```diff
@@ -82,3 +83,25 @@ func performValidation(ctx context.Context, state *tfsdk.State, plan *tfsdk.Plan
 		diags.AddError(fmt.Sprintf("`%s` %s", validationPath, v.Description(ctx)), "")
 	}
 }
+
+type RegionSpecPriorityOrderDecreasingValidator struct{}
+
+func (v RegionSpecPriorityOrderDecreasingValidator) Description(ctx context.Context) string {
+	return v.MarkdownDescription(ctx)
+}
+func (v RegionSpecPriorityOrderDecreasingValidator) MarkdownDescription(_ context.Context) string {
+	return "must be a list with priority in descending order"
+}
+func (v RegionSpecPriorityOrderDecreasingValidator) ValidateList(ctx context.Context, req validator.ListRequest, resp *validator.ListResponse) {
```

Comment:
in these cases i find useful to also have unit tests with more test scenarios

Reply:
As this was a copy-paste from previous implementation it didn't seem that criticial